Model/Queue.php

```diff
@@ -82,7 +82,7 @@ public function dequeue()
          */
         $message = $this->createConsumer()->receive(self::TIMEOUT_PROCESS);
         if (null !== $message) {
-            $envelope = $this->createEnvelop($message);;
+            $envelope = $this->createEnvelop($message);
             return $envelope;
         }
         return null;
```

Comment:
Write like this:
```php
return $message ? $this->createEnvelop($message) : null;
```

Reply:
Thanks! I was just checking the code and saw the typo and did not check actual code. This is way better and changed it.